Dockerfile

```diff
@@ -1,23 +1,27 @@
 FROM ubuntu:14.04
 
+ENV SHELL="/bin/bash"
+
 RUN apt-get update && \
-    apt-get install -y wget \
-                       curl \
-                       software-properties-common \
+    apt-get install -y default-jdk \
                        git \
-                       unzip \
                        postgresql \
-                       default-jdk
+                       unzip \
```

Comment:
Really doesn't make a difference, but if you like, the Github API also offers tarball archives, (and I think you can safely expect `tar` to be pre-installed), _e.g._ –

    wget -O - https://api.github.com/repos/dssg/acs2pgsql/tarball/d25c8dc7c231645f70a36b4e1fd4466c7e9a8ed1 | tar -xzf -

(You might want `tar --strip-components=1 …` to remove an encapsulating directory though I don't think so.)

Alternatively, so long as you've got Git, `git archive` works well –

    git archive --remote=git@github.com:dssg/acs2pgsql.git d25c8dc7c231645f70a36b4e1fd4466c7e9a8ed1 | tar -xf -

Reply:
the drake script calls unzip, so I need to install it anyway (without more substantial refactoring)